external/webserver/http_client.c

```diff
@@ -197,7 +223,7 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url,
 					} else {
 						HTTP_LOGD("unknown version\n");
 					}
-				} else { /* If this is called by webclient */
+				} else if(response) { /* If this is called by webclient */
```

Comment:
could you add a space after "else if" please?

Reply:
If response is null, is this function working fine? Or, you should add null check & return routine regarding parameters at the start of function.
Especially, response referenced by this function in many places.

Reply:
response is NULL only when this function called from webserver. but in webserver case client is not NULL so this block will not hit. response will never be NULL when this methiod called from webclient.